README.md

```diff
@@ -67,6 +86,103 @@ dependencies {
+To use the driver with an IDE (for example, IntelliJ), download the `.jar` file and copy it into your `libs` folder.  Then, add the file to your CLASSPATH: 
+
+**Example - IntelliJ**
+
+```bash
+wget https://github.com/awslabs/aws-mysql-jdbc/releases/download/0.4.0/aws-mysql-jdbc-0.4.0.jar
+cp aws-mysql-jdbc-0.4.0.jar /home/userx/libs/
+export CLASSPATH=$CLASSPATH:/home/userx/libs/aws-mysql-jdbc-0.4.0.jar
+```
+
+After downloading the `.jar` file and updating the CLASSPATH, add the driver information to your `Project`:
+
+1. Open `Project Structure` in the IntelliJ user interface.
+
+2. Select `Modules`.
+
+3. Open the `Dependencies` tab for your project.
+
+4. Add the AWS JDBC Driver for MySQL .jar file.
+
+5. Update the connection URL prefix from jdbc:mysql:// to jdbc:mysql:aws://.
+
+**Upgrading to a newer version with IntelliJ**
+
+To upgrade to a newer version of the driver, download the updated driver, and add it to your CLASSPATH:  
+
+```bash
+wget https://github.com/awslabs/aws-mysql-jdbc/releases/download/0.4.0/aws-mysql-jdbc-0.4.0.jar
+cp aws-mysql-jdbc-0.4.0.jar /home/userx/libs/
+export CLASSPATH=$CLASSPATH:/home/userx/libs/aws-mysql-jdbc-0.4.0.jar
+```
+
+Then, follow the steps listed above to update your project to the latest version.
+
+
+#### Using the Driver with a database client (DBeaver)
+To use the driver with the DBeaver database client, download the `.jar` file, copy the file to your `libs` folder, and add the file to your `CLASSPATH`:
+
+**Example - DBeaver**
+
+```bash
+wget https://github.com/awslabs/aws-mysql-jdbc/releases/download/0.4.0/aws-mysql-jdbc-0.4.0.jar
+cp aws-mysql-jdbc-0.4.0.jar /home/userx/libs/
+export CLASSPATH=$CLASSPATH:/home/userx/libs/aws-mysql-jdbc-0.4.0.jar
+```
+
+After downloading the .jar file and adding it to your CLASSPATH, add the driver information to the DBeaver client:
+
+1. Open DBeaver’s `Driver Manager` window and click `New` to add the AWS JDBC Driver.
+
+2. In `Settings`, enter the following information:
+
+```bash
+Default Port: 3306
+Driver Name: AWS JDBC Driver for MySQL
+Class Name: software.aws.rds.jdbc.mysql.Driver
+URL Template: jdbc:mysql:aws://<endpoint>
+```
+
+3. Go to the `Libraries` tab, click `Add Artifact` and add the following dependency declaration:
+
+```bash
+<dependency>
+  <groupId>software.aws.rds</groupId>
+  <artifactId>aws-mysql-jdbc</artifactId>
+</dependency>
+```
+
+4. DBeaver will respond: `Artifacts Found: 1`.
+
+5. Click `OK` to finish adding the driver.
+
+After adding driver information, you can create new connections that use the AWS JDBC Driver for MySQL.
+
+**Upgrading to a newer version with the DBeaver Client**
+
+To upgrade to a newer version of the driver, download the updated driver, and add it to your CLASSPATH:  
+
+```bash
+wget https://github.com/awslabs/aws-mysql-jdbc/releases/download/0.4.0/aws-mysql-jdbc-1.0.0.jar
+cp aws-mysql-jdbc-1.0.0.jar /home/userx/libs/
+export CLASSPATH=$CLASSPATH:/home/userx/libs/aws-mysql-jdbc-0.4.0.jar
```

Comment:
Versioning 0.4.0 -> 1.0.0
& file path is a mix of 0.4.0 & 1.0.0

Reply:
Yes, this would be right. However, if we're merging in these changes before our release, let's keep this at 0.4.0. We'll just have to bump up to 1.0.0 when the release happens.

Reply:
Thanks for clarifying, this makes sense to me. Thanks Susan!